stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/decorator/ThreadRepliesDecorator.kt

```diff
@@ -138,6 +141,37 @@ internal class ThreadRepliesDecorator : BaseDecorator() {
             getRepliesQuantityString(rootView.resources, replyCount)
     }
 
+    private fun revertDefaultFootnoteConstraints(
+        rootView: ConstraintLayout,
+        footnoteId: Int,
+        threadRepliesFootnoteId: Int,
+        footnoteAnchorViewId: Int,
+        data: MessageListItem.MessageItem
+    ) {
+        rootView.updateConstraints {
+            clearVerticalConstraints(footnoteId, threadRepliesFootnoteId)
+            connect(footnoteId, ConstraintSet.TOP, footnoteAnchorViewId, ConstraintSet.BOTTOM)
+            clearHorizontalConstraints(footnoteId, threadRepliesFootnoteId)
+            if (data.isTheirs) {
+                connect(footnoteId, ConstraintSet.LEFT, footnoteAnchorViewId, ConstraintSet.LEFT)
+                connect(threadRepliesFootnoteId, ConstraintSet.LEFT, footnoteId, ConstraintSet.RIGHT)
+            } else {
+                connect(
+                    footnoteId,
+                    ConstraintSet.RIGHT,
+                    footnoteAnchorViewId,
+                    ConstraintSet.RIGHT
+                )
+                connect(threadRepliesFootnoteId, ConstraintSet.RIGHT, footnoteId, ConstraintSet.LEFT)
+            }
```

Comment:
Just a small note: Perhaps some of this could be simpler by using biases instead of clearing and then reconnecting constraints. There's an example of that in `GravityDecorator`.

Reply:
I'll merge the fix now, and simplify it in the next iteration